dashboard/src/composables/format.ts

```diff
@@ -0,0 +1,34 @@
+import moment from "moment";
+
+export function FormatDateTime(value: Date | undefined) {
```

Comment:
I think that all these functions that you're exporting should use camelCase.
PascalCase is used for classes, components, types...

Reply:
@sevein are you referencing a particular Typescript style guide? I don't think we have any defined conventions for our Typescript / Vue code.